fearless/Modules/Crowdloan/CrowdloanContribution/CrowdloanContributionInteractor.swift

```diff
@@ -69,12 +70,16 @@ class CrowdloanContributionInteractor: CrowdloanContributionInteractorInputProto
             self?.presenter.didReceiveLeasingPeriod(result: result)
         }
 
-        fetchConstant(
-            for: .existentialDeposit,
-            runtimeCodingService: runtimeService,
-            operationManager: operationManager
-        ) { [weak self] (result: Result<BigUInt, Error>) in
-            self?.presenter.didReceiveMinimumBalance(result: result)
+        if chainAsset.chain.isOrml {
+            presenter?.didReceiveMinimumBalance(result: .success(BigUInt.zero))
```

Comment:
It's not zero in result?

Reply:
Yeah should work conforming to ExpressibleByIntegerLiteral or something like that